README.md

```diff
@@ -75,7 +89,16 @@ the variable pizza inside your method.
 
 Your Answer:
 ```js
-// write code here
+Objects are collections of key-value pair properties.
+
+var pizza = {
+  temperature: 70,
+  bake: function (){
+    temperature = 300;
+    console.log("the pizza's temp is " + temperature)
+  }
+}
+pizza.bake();
 ```
```

Comment:
You need to reference the temperature using `this.temperature` inside of `bake`.